src/main/java/forestry/apiculture/ApiaristAI.java

```diff
@@ -0,0 +1,152 @@
+package forestry.apiculture;
+
+import forestry.api.apiculture.EnumBeeType;
+import forestry.apiculture.blocks.BlockApiculture;
+import forestry.apiculture.items.ItemBeeGE;
+import forestry.apiculture.tiles.TileBeeHouse;
+import forestry.core.tiles.TileUtil;
+import forestry.core.utils.InventoryUtil;
+import net.minecraft.block.Block;
+import net.minecraft.entity.ai.EntityAIMoveToBlock;
+import net.minecraft.entity.passive.EntityVillager;
+import net.minecraft.inventory.InventoryBasic;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+@SuppressWarnings("static-access")
+public class ApiaristAI extends EntityAIMoveToBlock{
+	private final EntityVillager villager;
+	private boolean hasDrone;
+	private boolean hasPrincess;
+	private InventoryBasic villagerInventory;
+	
+	public ApiaristAI(EntityVillager villager, double speed) {
+		super(villager, speed, 16);
+		this.villager = villager;
+		villagerInventory = villager.getVillagerInventory();
+	}
+	
+	@Override
+	public boolean shouldExecute() {
+		if(this.runDelay<0) {
+			this.hasDrone=hasBeeType(EnumBeeType.DRONE);
+			this.hasPrincess=hasBeeType(EnumBeeType.PRINCESS);
+		}
+		return super.shouldExecute();
+	}
+	
+	@Override 
+	public void updateTask()
+	 {
+        super.updateTask();
+        this.villager.getLookHelper().setLookPosition((double)this.destinationBlock.getX() + 0.5D, (double)(this.destinationBlock.getY() + 1), (double)this.destinationBlock.getZ() + 0.5D, 10.0F, (float)this.villager.getVerticalFaceSpeed());
+
+        if (this.villager.getDistanceSqToCenter(this.destinationBlock) > 4.0D)
+        {
```

Comment:
one file should not have multiple brace styles. please use the brace style of this project